generator.rb

```diff
@@ -0,0 +1,111 @@
+def is_vowel(letter)
+  case letter
+  when "a", "e", "i", "o", "u"
+    return "true"
+  else
+    return "false"
```

Comment:
You should return `true` or `false` (the boolean value) here, rather than `"true"` or `"false"` (a string containing the word "true" or "false"). Getting comfortable with using different data types is an important thing to practice early on.